src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -536,22 +538,92 @@ def __init__(
         self.__producer_topic = settings.KAFKA_TOPICS[settings.KAFKA_SNUBA_METRICS].get(
             "topic", "snuba-metrics"
         )
+        self.__commit_function = commit_function
 
         self.__closed = False
         self.__metrics = get_metrics()
+        self.__produced_message_offsets = {}
+        self.__callbacks = 0
+        self.__started = time.time()
+        # Need to make these flags
+        self.__commit_max_batch_size = 10000
+        self.__commit_max_batch_time = 4.0
```

Comment:
Let's start with 25k and 10 seconds. Just to ensure we move the commit time out of the way when we look at our perf